index.html

```diff
@@ -4674,23 +4674,199 @@ <h2>
-<a href="#did-controller"></a>. Not all <a>DID method</a>
-specifications will recognize control from <a>DIDs</a> registered using other
-<a>DID methods</a> and they might restrict third-party control to <a>DIDs</a>
-that use the same method. Access control and key recovery in a
-<a>DID method</a> specification can also include a time lock feature to
-protect against key compromise by maintaining a second track of control for
-recovery.
-</p>
+        Section <a href="#method-operations"></a> specifies the
+        <a>DID</a> operations to be supported by a
+        <a>DID method</a> specification, including
+        <a href="#update">update</a>
+        which MUST be used to perform a verification method rotation.
+      </p>
+  
+      <p>
+        A controller performs a rotation when they add a new verification
+        method that is meant to replace an existing verification method after
+        some time.
+      </p>
+  
+      <p>Not all DID Methods support verification method rotation.</p>
+  
+      <p>
+        Unlike revocation, rotation allows for valid new verifications, 
+        while invalidating older verifications after the older verification method is removed
+        from the latest DID Document.
+      </p>
+  
+      <p>
+        Rotation is a useful mechanism for protecting against verification
+        method compromise, since frequent rotation of a verification method by
+        the controller, reduces the value of a compromised verification method
+        to an attacker.
+      </p>
+  
+      <p class="note">
+        Higher security environments tend to employ more frequent verification
+        method rotation.
+      </p>
+  
+      <p class="note">
+        Frequent rotation of a verification method might be frustrating for
+        parties that are forced to continiously renew or refresh associated
+        credentials.
+      </p>
+    </section>
+  
+    <section>
+      <h2>Verification Method Revocation</h2>
+  
+      <p>Verification method revocation is a reactive security measure.</p>
+
+      <p>
+        Verification method revocation applies only to the current or latest
+        verison of a DID Document.
+      </p>
+  
+      <p>
+        If a verification method is no longer exclusively accessible to the
+        controller or parties trusted to act on behalf of the controller, it
+        should be revoked immediatly to reduce the risk of masquerading, theft
+        and fraud.
+      </p>
+  
+      <p class="advisement">
+        It is considered a best practice to support key revocation.
+      </p>
+  
+      <p class="advisement">
+        A controller should revoke any verification method that is believed to
+        be compromised immediatly.
+      </p>
+  
+      <p class="advisement">
+        Cryptographic verifications associated with revoked verification
+        methods should be considered invalid regardless of when revocation
+        occurs.
```

Comment:
@dhh1128, could you make a suggestion here? I presume you'd like this language tweaked to be more clear that verifiers may decide to continue to honor proofs previously made and verified in the past.

Reply:
yes, in essence, I think he wants to allow verifiers to ignore that keys have been revoked... which is their prerogative.